deis/cloud-config.yaml

```diff
@@ -0,0 +1,178 @@
+#cloud-config
+---
+coreos:
+  etcd:
+    # generate a new token for each unique cluster from https://discovery.etcd.io/new
+    # uncomment the following line and replace it with your discovery URL
+    discovery: https://discovery.etcd.io/402d2ffba1539292992aff540c8dc3d9
```

Comment:
its a nice-to-have but myself and customers forget to do this step all the time. and when you do forget you don't find out until your cluster is provisioned. doesn't take much effort. at the very least we should change the hex to `<discovery hex>` or something.